tests/e2e/tailscale/Vagrantfile

```diff
@@ -45,13 +45,25 @@ def provision(vm, roles, role_num, node_num)
     end
   end
   if roles.include?("agent")
+    vpn_auth = nil
+    vpn_auth_method = nil
+    auth_info = "name=tailscale,joinKey=#{TAILSCALE_KEY}"
+    if role_num == 0
+      vpn_auth_method = "vpn-auth"
+      vpn_auth = auth_info
+    else
+      vpn_auth_method = "vpn-auth-file"
+      File.write("./vpn-auth-file", auth_info)
+      vm.provision "file", source: "./vpn-auth-file", destination: "$HOME/vpn-auth-file"
+      vpn_auth = File.expand_path("$HOME/vpn-auth-file")
```

Comment:
The problem with this is that in line 57 $HOME is "/home/vagrant" and in line 58, $HOME is your node's home, so you end up with a config pointing to a path which does not make sense inside the vagrant VM. Probably the best solution is to use "/home/vagrant/" in line 58 instead of $HOME